### PR TITLE
tutorial6: cone orientation

### PR DESCRIPTION
**Pull request by jngrad**
Fixes cone issue in #2129

### CODE REVIEW

File: doc/tutorials/06-active_matter/SOLUTIONS/rectification_simulation.py [hunk 1]
```diff
@@ -82,20 +83,20 @@ def a2quat(phi,theta):
...
+prod_steps = 500
 prod_length = 500
-dt          = 0.01
+dt = 0.01
 
 # Setup the MD parameters
 
 system = espressomd.System(box_l=[length, diameter + 4, diameter + 4])
-system.set_random_state_PRNG()
+system.box_l = [length, diameter + 4, diameter + 4]
```
Comment (mkuron) [inline doc/tutorials/06-active_matter/SOLUTIONS/rectification_simulation.py]: You deleted the RNG seeding and set the box length twice. Also in some of the other files.

File: doc/tutorials/06-active_matter/SOLUTIONS/rectification_geometry.py [hunk 2]
```diff
@@ -49,13 +49,13 @@
...
-length   = 100
+length = 100
 diameter = 20
-dt       = 0.01
+dt = 0.01
 
 # Setup the MD parameters
 
-system = espressomd.System(box_l=[length, diameter+4, diameter+4])
+system = espressomd.System(box_l=[length, dieameter + 4, diameter + 4])
```
Comment (mkuron) [inline doc/tutorials/06-active_matter/SOLUTIONS/rectification_geometry.py]: typo in `diameter`. You didn't test the scripts after making your changes.

File: doc/tutorials/06-active_matter/EXERCISES/rectification_geometry.py [hunk 3]
```diff
@@ -36,11 +36,9 @@
 from espressomd.shapes import Cylinder, Wall, HollowCone
 
 
-assert_features(["LB_GPU","LB_BOUNDARIES_GPU"])
-
 # Setup constants
 
-outdir = "./RESULTS_RECTIFICATION_GEOMETRY/"
+outdir = "./RESULTS_RECTIFICATION/"
```
Comment (mkuron) [inline doc/tutorials/06-active_matter/EXERCISES/rectification_geometry.py]: Undo this change. Right now the template and the solution don't match.

File: doc/tutorials/06-active_matter/EXERCISES/rectification_geometry.py [hunk 4]
```diff
@@ -36,11 +36,9 @@
 from espressomd.shapes import Cylinder, Wall, HollowCone
 
 
-assert_features(["LB_GPU","LB_BOUNDARIES_GPU"])
-
```
Comment (mkuron) [inline doc/tutorials/06-active_matter/EXERCISES/rectification_geometry.py]: Restore this, I think it's still valid.

File: doc/tutorials/06-active_matter/SOLUTIONS/rectification_simulation.py [hunk 5]
```diff
@@ -30,46 +30,47 @@
...
 from espressomd.shapes import Cylinder, Wall, HollowCone
-from espressomd.lbboundaries import LBBoundary
 
-assert_features(["ENGINE","LENNARD_JONES","ROTATION","MASS"])
+
+assert_features(["ENGINE", "CONSTRAINTS", "LENNARD_JONES", "ROTATION", "MASS"])
 
 # Quaternion procedure
 
 def a2quat(phi,theta):
```
Comment (mkuron) [inline doc/tutorials/06-active_matter/SOLUTIONS/rectification_simulation.py]: This doesn't belong here.

File: tools/blockfile.py [hunk 6]
```diff
@@ -1,225 +1,225 @@
...
+    # extracts the list of column types from scripts/blockfile_support.tcl.
+    if len(sys.argv) > 1 and sys.argv[1] == '--extract':
+        if len(sys.argv) < 2 or not os.path.exists(sys.argv[2]):
+            sys.stderr.write(
+                "Please specify the path to scripts/blockfile_support.tcl from Espresso.\n")
+            sys.exit(1)
+        load_col_types(sys.argv[2])
+        from pprint import pprint
+        print(pprint(col_types))
+        sys.exit()
```
Comment (mkuron) [inline tools/blockfile.py]: What happened here?

**Review by mkuron — CHANGES_REQUESTED** [review]

**Comment by jngrad** [comment]
Something went wrong when merging with upstream, I reverted it. The [branch diff](https://github.com/jngrad/espresso/compare/517170998a6684863...jngrad:d482f419577d32e0193add) should be clean now.

Side note: `tools/blockfile.py` has Windows line feeds (CRLF) instead of Unix line feeds (LF), this can be confusing when doing merges.

**Comment by mkuron** [comment]
A reverted merge will actually revert all changes that happened between the branch and the merge, which is bad. Please do the following after ensuring that you no uncomitted changes:
```
git reset --hard 7bb569a
git push -f
```
This will delete both the merge and its reversal.

**Comment by mkuron** [comment]
I can retry the merge myself tomorrow. Also, the CRLFs are definitely there in error.

**Comment by jngrad** [comment]
Interesting, after the reset only the 5 tutorial files we were working on are in conflict, instead of the whole project. Also, the tutorial files have line feed issues too?

**Comment by jngrad** [comment]
@mkuron I've merged `python` into the proposed PR in a [new branch](https://github.com/jngrad/espresso/compare/pr1...jngrad:pr1-merge) to solve the conflicts, let me know if you still want to retry this merge yourself

**Comment by mkuron** [comment]
Yes, that new merge looks good. Please push it to this pull request‘s branch. If CI passes, @RudolfWeeber can go ahead and merge.

**Comment by RudolfWeeber** [comment]
I learned about 
git merge -Xignore-space-change
while having a look.

**Comment by jngrad** [comment]
running fine

**Comment by jngrad** [comment]
interesting: if you mix CRLF and LF line feeds in the same file during a merge between a CRLF file and a LF file, the continuous integration command `git --no-pager diff HEAD^ HEAD > style.patch` produces a patch to fix the line feeds, but when you download it, the file is sanitized to LF line feeds, making it useless

**Pull request merged**